src/App.tsx

```diff
@@ -1,24 +1,295 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useEffect,
+  useState,
+  useMemo,
+  useRef,
+} from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import {
+  deleteTodoById,
+  getTodos,
+  setNewTodo,
+  updateTodoById,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { Notification } from './components/Notification/Notification';
+import { StatusEnum } from './types/StatusEnum';
+import { TodosFilter } from './components/TodosFilter/TodosFilter';
+import { TodoList } from './components/TodoList/TodoList';
 
-const USER_ID = 0;
+const USER_ID = 11497;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isLoadingAddTodo, setIsLoadingAddTodo] = useState(false);
+  const [todosIdToDelete, setTodosIdToDelete] = useState<number[]>([]);
+  const [todosIdToUpdate, setTodosIdToUpdate] = useState<number[]>([]);
+  const [filter, setFilter] = useState<StatusEnum>(
+    window.location.hash.slice(2) as StatusEnum || StatusEnum.All,
+  );
+
+  const formInput = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (!todosIdToDelete.length && !isLoadingAddTodo && formInput.current) {
+      formInput.current.focus();
+    }
+  }, [isLoadingAddTodo, todosIdToDelete]);
+
+  const handleChangeFilter = (newFilter: StatusEnum) => {
+    setFilter(newFilter);
+  };
+
+  const handleChangeNewTodoTitle
+    = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleChangeNewTodoTitle = (
    event: React.ChangeEvent<HTMLInputElement>
  ) => {
```
Don't forget about formating